docker-compose.yml

```
@@ -0,0 +1,19 @@
+services:
+  db:
+    image: mysql:8.0
+    container_name: ticha-db
+    restart: always
+    environment:
+      MYSQL_ROOT_PASSWORD: ${MYSQL_ROOT_PASSWORD}
+      MYSQL_DATABASE: ${MYSQL_DATABASE}
+      MYSQL_USER: ${MYSQL_USER}
+      MYSQL_PASSWORD: ${MYSQL_PASSWORD}
+    ports:
+      - ${MYSQL_PORT}:3306  
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)